.github/workflows/ci-vulnerability-scans.yml

```
@@ -0,0 +1,98 @@
+# GitHub Actions CI workflow that runs vulnerability scans on the application's Docker image
+# to ensure images built are secure before they are deployed. 
+
+# NOTE: The workflow isn't able to pass the docker image between jobs, so each builds the image.
+#       A future PR will pass the image between the scans to reduce overhead and increase speed
+name: API Linting & Testing CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - app/**
+  pull_request:
+    paths:
+        - app/**
```

nit, this line is indented differently
```suggestion
      - app/**
```